packages/components/src/theme/grid.scss

```diff
@@ -239,6 +237,8 @@ $table-cell-border: $table-cell-border-width solid transparent;
 $table-cell-selected-border: $table-cell-border-width solid #2980b9;
 $table-cell-selected-box-shadow: 0 3px 3px darkgrey;
 $table-cell-selection-bg-color: #EDF3FF;
+$table-cell-max-height: 200px;
```

Comment:
These are obviously arbitrary but I'm looking to constrain how wide/high a cell can be as it can really diminish the user experience if they have to scroll excessively to reach the next row/column. That said, the implementation currently cuts off the rest of the cell content but we could consider exposing scrolling within a cell (or just not having these limits).

Reply:
In testing this out locally a bit, I agree that setting a max height is a good idea. The scrolling within the cell once this height is reached works well and seems better than letting it expand to the full height.